portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +28,33 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+var slideIndex = 1;
+showSlides(slideIndex);
+
+// Next/previous controls
+function plusSlides(n) {
+  showSlides(slideIndex += n);
+}
+
+// Thumbnail image controls
+function currentSlide(n) {
+  showSlides(slideIndex = n);
+}
+
+function showSlides(n) {
+  var i;
+  var slides = document.getElementsByClassName("mySlides");
+  var dots = document.getElementsByClassName("demo");
```

Comment:
Ok I'm not entirely sure what dots is supposed to be but it's possible I'm not looking at this carefully enough. I also don't see a "demo" class/tag? Could you please offer some clarification (maybe in the naming of the variable).

Reply:
+1

Reply:
Okay sorry about that renamed the class tag. It just adds a transparency hover effect in the css, class called "thumbimagecl".